routes/componentRoutes.js

```diff
@@ -35,32 +33,34 @@ router.get('/:type/:id', async (req, res) => {
  * route to patch the components in a section
  * @param {string} sectionId
  */
+//this function was deleting components unintentionally, for now it's deprecated. 
 router.patch('/:sectionId', async (req, res) => {
```

Comment:
If we are deprecating (dont you mean temporarily disable? deprecation doesnt disable functionality) this we need to handle it properly, if someone was to use this endpoint they would get an _OK_ response and think that they modified a resource also a headache for other devs that could rely on this and now think their code is broken.
This endpoint should return 503 Service unavailable https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503